<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>base</class>
 <widget class="QWidget" name="base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>24</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="layout_main">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label_source">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_source"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_source">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_dest">
     <property name="text">
      <string>Destination:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="edit_dest"/>
   </item>
   <item>
    <widget class="QPushButton" name="button_delete">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../logos/logo_cross.png</normaloff>../logos/logo_cross.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
